test/e2e/generator/generate.go

```diff
@@ -34,7 +34,7 @@ var (
 	}
 
 	// The following specify randomly chosen values for testnet nodes.
-	nodeDatabases = uniformChoice{"goleveldb", "cleveldb", "rocksdb", "boltdb", "badgerdb", "pebbledb"}
+	nodeDatabases = uniformChoice{"goleveldb", "rocksdb", "badgerdb", "pebbledb"}
```

Comment:
Does it make sense to have `goleveldb` here ? In the Dockerfile (above) seems we don't build images with these option.

Reply:
CometBFT comes with goleveldb. You don't need a flag for it.